spec/features/users/user_signs_in_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+feature "user signs in", %{
+  As a signed up user
+  I want to sign in
+  So that I can regain access to my account
+} do
+  scenario specify valid credentials do
+    user = FactoryGirl.create(:user)
+
+    visit new_user_session_path
+
+    fill_in Email, with: user.email
+    fill_in Password, with: user.password
+
+    click_button "Log in"
```

Comment:
Tyler wrote a login helper to do this. Talk to him about how this test can be refactored.